src/main/scala/viper/gobra/frontend/info/implementation/typing/StmtTyping.scala

```diff
@@ -136,6 +136,9 @@ trait StmtTyping extends BaseTyping { this: TypeInfoImpl =>
     case _: PBlock => noMessages
     case _: PSeq => noMessages
     case _: PEmptyStmt => noMessages
+    case _: PGoto => noMessages
+    case _: PContinue => noMessages
+    case _: PBreak => noMessages
```

Comment:
For `continue` and `break`, we should add checks here. In particular, `continue` statements can only appear in loops and `break` can only appear in loops, switch statements and select statements.

Reply:
I think the idea was that only parser support is added for now and nothing else

Reply:
Sure, but then I would replace `noMessages` with either `???` or a violation such that we do not forget in the future to add this.

Reply:
I've changed it to `???` for now.